Adyen/Core/Components/AbstractPersonalInformationComponent/AbstractPersonalInformationComponent.swift

```diff
@@ -210,4 +225,39 @@ open class AbstractPersonalInformationComponent: PaymentComponent, PresentableCo
         payment?.countryCode ?? Locale.current.regionCode ?? "US"
     }
 
+    /// :nodoc:
+    public func showValidation() {
+        formViewController.showValidation()
+    }
+
+    /// :nodoc:
+    internal func populateFields() {
+        guard let shopperInformation = shopperInformation else { return }
+
+        if let firstName = shopperInformation.shopperName?.firstName {
```

Comment:
Do we need if lets for these? 
Trying to understand myself too, what happens if we just go `firstNameItem?.value  = shopperInformation.shopperName?.firstName` ?

Reply:
`value` is non-optional type so we need to unwrap it somehow. I was thinking that maybe something like this may look better instead of having `if let` unwrapping. What do you think?

```swift
shopperInformation.shopperName.map {
    firstNameItem?.value = $0.firstName
    lastNameItem?.value = $0.lastName
}
```

Reply:
Ah my bad, when I saw items optional, I assumed value is too.
That map looks very nice indeed!